### ISSUE TITLE
Ningaloo Coast eDNA feedback

### ISSUE THREAD

**Issue by obishelpdesk** [iobis/edna-species-lists#24]
On behalf of sallyann.gudge@dbca.wa.gov.au

The results are comprehensive and the report is professional and comprehensive.  DBCA staff enjoyed undertaking the research (although the logistics were a bit challenging wrt organising community participation).  The Traditional Owners (Baiyungu Rangers) found it interesting.  We compared the data to the 2023 year of Diver Operatoed Video (DOV) fish data and we observed 220 species, so it is outperforming a single method in terms of diversity. With the DOVs and BRUVs combined though we would exceed what this eDNA has detected.  eDNA is less effort, but DBCA will be doing DOVs and BRUVs anyway, and eDNA will not be able to replace either until the method is able to provide the ability to quantify things (abundance and biomass). The one area that I would say that this data could be valuable is to see if there is less variation in the species richness results temporally, compared to DOVs.  We get too much variation to reliably track species richness over time with just one method (i.e. DOVs) so this could be a valuable addition, especially if we don't have to pay for the surveys or analysis as DBCA (and I imagine other marine WHA's) are very unlikely to be able to afford or justify this expense for just one metric.

So, to summarise, for our current monitoring of finfish we are unlikely to use this data, but repeated sampling across years may provide some insight into the potential of eDNA for temporal monitoring of species richness. Given this is just one metric though, we are unlikely to invest in eDNA sampling just yet.  DBCA would be happy to discuss the application of eDNA research across years and its value for use in future research for Ningaloo WHA and other marine tenure (eg the Exmouth Gulf proposed marine park).
